Sync main

brianwdavis opened the pull request:
Bring in onfarm-api changes

schatterje22 committed:
Add gitignore

schatterje22 committed:
Adding secrets to gitignore

schatterje22 committed:
Setting up to use the onfarm data in Planet API

schatterje22 committed:
Download orders

schatterje22 committed:
error checking,
onfarm visulization

brianwdavis committed:
Merge pull request #2 from precision-sustainable-ag/Features/onfarm

PSA on farm data with Planet API

Hc2112 committed:
Update Hidalgo_PA_planet_orders.R

added brief explanations for some objects and edited product.order.name, downloads, and scenes objects.

jjennewein reviewed (changes requested):
looks good, but I don't see the change to also have 4 band as a possible query... I see new annotation on line 84 that still uses 8band. Please also add the 4band option in here

Hc2112 committed:
Update Hildalgo_PA_planet_orders.R

edited products.json and added products object for choosing between 8-band and 4-band surface reflective products.

Hc2112 committed:
Update Hildalgo_PA_planet_orders.R

added filter where quality_category equals "standard". Standard images have off-nadir angles <20, etc.

Hc2112 committed:
Create Planet Metadata.r

Hc2112 committed:
Update Hildalgo_PA_planet_orders.R
